f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_funding_plan_schedule_text.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update>
+    <sys_dictionary action="INSERT_OR_UPDATE" element="schedule_text" table="x_g_dis_atat_funding_plan">
+        <active>true</active>
+        <array>false</array>
+        <array_denormalized>false</array_denormalized>
+        <attributes/>
+        <audit>false</audit>
+        <calculation><![CDATA[(function calculatedFieldValue(current) {
+	var newline = '\n';
+	var increments = current.getValue('remaining_amount_increments').split(",");
+	var ouputArray = [];
+	for (var i = 0; i < increments.length; i++) {
+      var gr = new GlideRecord('x_g_dis_atat_funding_increment');
+      gr.addQuery('sys_id=' + increments[i]);
+      gr.query();
+      while(gr.next()) {
+        // format like 'Funding Increment #X:\nX QTR FYXX - $XX'
+        var line = 'Funding Increment #' + gr.getValue('order') + ":" + newline;
+        line += gr.getValue('description') + ' - ' + gr.getDisplayValue('amount');
+        ouputArray.push(line);
+      }
```

Comment:
You already mentioned you would do this, but I already made the comment. I think this is how it would be done using `.get()` from glideRecord.

```suggestion
      gr.get(increments[i]);
        // format like 'Funding Increment #X:\nX QTR FYXX - $XX'
        var line = 'Funding Increment #' + gr.getValue('order') + ":" + newline;
        line += gr.getValue('description') + ' - ' + gr.getDisplayValue('amount');
        ouputArray.push(line);
```